pep-0654.rst

```diff
@@ -821,18 +822,19 @@ chaining:
       +-+---------------- 1 ----------------
         | ExceptionGroup
         |    with one sub-exception:
-        +-+---------------- 1.1 ----------------
+        +-+---------------- 1.cause.1 ----------------
           | Traceback (most recent call last):
           |   File "<stdin>", line 2, in <module>
           | TypeError: bad type
-
+        +------------------------------------
+        |
         | The above exception was the direct cause of the following exception:
-
+        |
         | Traceback (most recent call last):
         |   File "<stdin>", line 4, in <module>
         | ValueError: bad value
       +------------------------------------
```

Comment:
Shouldn't this one be aligned with what's above it? (Also in a few other places.)

There's also a redundant extra line of dashes above on line 794 (GitHub won't let me comment there).

Reply:
I think so too, I probably misunderstood what you meant earlier. Online 802, it should be indented 4 more places so the + is under the |, right?

Reply:
Exactly!